compare.py

```diff
@@ -20,11 +21,12 @@ def prepare_for_comparison(frame_dict):
   A single DataFrame in the same format as returned by setup_individual_obs_df but with an additional
   run_name column.
   """
```

Comment:
Hmm, this seems a little weird... I would've expected black to bump this whole comment text block over to a 4-space tab, including the closing triple-quote.

I see this pattern repeats.